config/defaults.mk

```diff
@@ -143,6 +143,7 @@ MFEM_USE_RAJA          = NO
 MFEM_USE_OCCA          = NO
 MFEM_USE_CEED          = NO
 MFEM_USE_UMPIRE        = NO
+MFEM_USE_CAMP          = NO
```

Comment:
Is CAMP now a requirement? In which cases?

Either way, it should probably be documented in INSTALL

Reply:
It's required for recent Umpire releases, I'll add a note to INSTALL